libs/platform/src/lib/components/list/list.component.html

```diff
@@ -35,4 +36,7 @@
 </ng-template>
 <ng-template #elseBlock>
     <ng-content></ng-content>
-</ng-template>
\ No newline at end of file
+</ng-template>
+
+<span class="fd-hidden" id='loading' i18n="@@list.loading">loading</span> 
+<span class="fd-hidden" id='loadingtitle' i18n="@@list.loadingtitle">{{loadTitle}}</span>
```

Comment:
let's create more unique ids to avoid potential collisions with other elements in the document